### PR TITLE
fix: Consul health check runtime config

### PR DESCRIPTION
**Pull request by beaufrusetta**
Fixes https://github.com/edgexfoundry/developer-scripts/issues/242
Fixes https://github.com/edgexfoundry/developer-scripts/issues/220
Fixes https://github.com/edgexfoundry/developer-scripts/issues/213

Added the new environment variables to the consul services in each of
the compose files. The vars, EDGEX_DB and EDGEX_SECURE are utilized in
an entrypoint script in the docker-edgex-consul image that initializes
default config and script files based on these environment variables.
These variables are NOT required (as they have defaults if the system
can't find them), however, it is encouraged to use them so that
configuration is explicitly set, and the developers happen to know what
might be going on in the event of a misconfigured consul service.

Signed-off-by: Beau Frusetta <beau.frusetta@intel.com>

### CODE REVIEW

**Comment by beaufrusetta** [comment]
### Testing this PR
To test/verify this PR, please see the [testing instructions](https://github.com/edgexfoundry/docker-edgex-consul/pull/26#issuecomment-611041913) that I created in the PR comments for `docker-edgex-consul`.

File: releases/nightly-build/compose-files/docker-compose-nexus-mongo-no-secty-arm64.yml [hunk 1]
```diff
@@ -69,6 +69,9 @@ services:
       - consul-data:/consul/data:z
     depends_on:
       - volume
+    environment:
```
Comment (bnevis-i) [inline releases/nightly-build/compose-files/docker-compose-nexus-mongo-no-secty-arm64.yml:72]: Please change above image: tag to

```image: nexus3.edgexfoundry.org:10004/docker-edgex-volume-arm64:master```

File: releases/nightly-build/compose-files/docker-compose-nexus-redis-no-secty-arm64.yml [hunk 2]
```diff
@@ -66,6 +66,9 @@ services:
       - consul-data:/consul/data:z
     depends_on:
       - volume
+    environment:
```
Comment (bnevis-i) [inline releases/nightly-build/compose-files/docker-compose-nexus-redis-no-secty-arm64.yml:69]: Please change above `image` tag to

```
image: nexus3.edgexfoundry.org:10004/docker-edgex-volume-arm64:master
```

**Comment by bnevis-i** [comment]
PR got hosed by bad series of git commands... working with Beau to recover it.

File: releases/nightly-build/compose-files/docker-compose-nexus-mongo-no-secty-arm64.yml (hunk 1 above)
Comment (bnevis-i) [reply releases/nightly-build/compose-files/docker-compose-nexus-mongo-no-secty-arm64.yml:79]: meant -consul not -volume

File: releases/nightly-build/compose-files/docker-compose-nexus-redis-no-secty-arm64.yml (hunk 2 above)
Comment (bnevis-i) [reply releases/nightly-build/compose-files/docker-compose-nexus-redis-no-secty-arm64.yml:74]: typo mean -consul- not volume.

**Pull request merged**